lms/templates/commerce/provider.underscore

```diff
@@ -0,0 +1,13 @@
+<p class="bold_param">
+  <%= interpolate(gettext("You still need to see visit %s to complete the credit process."), [interpolate("<a href='%s'>%s</a>", [provider_url, display_name]) ]) %>
```

Comment:
Maybe I'm missing something in the design, but does this need to be a link?  The `provider_url` is a URL that we will POST to, but it doesn't necessarily support GET requests.